modules/optimization/test/tests/optimizationreporter/general_opt/tests

```diff
@@ -0,0 +1,14 @@
+[Tests]
```

Comment:
This is a good test, It took me a while to figure out what you were doing.  Should we just make the quadraticMinimize test like this?

Reply:
I think this is cleaner way because it allows to try different optimization options with recompiling or creating multiple test classes. It also is a good test to show how "general" GeneralOptimization is.